src/input/kbI2cBase.cpp

```diff
@@ -295,6 +295,7 @@ int32_t KbI2cBase::runOnce()
             case 0x9b: // fn+s      INPUT_BROKER_MSG_SHUTDOWN
             case 0xac: // fn+m      INPUT_BROKER_MSG_MUTE_TOGGLE
             case 0x9e: // fn+g      INPUT_BROKER_MSG_GPS_TOGGLE
+            case 0x8e: // fn+w      INPUT_BROKER_MSG_WAYPOINTFRAME_DISMISS
```

Comment:
Should we make something more generic to also dismiss text message frames as well?
Fn+D for delete / dismiss perhaps

Reply:
That could be a good call. Would you be happy for me to revert that combo for now, and address it separately in the near future?

Reply:
What about just switching the combo to Fn+D for now and continuing the rest of the PR as is? Then we can follow up with extending the functionality to the text message frames later. :-)

Reply:
I think it might be a bit late in the evening now to trust my brain with a quick push for this change, but I'll get back on it tomorrow, and might as well figure out the logistics of the general-dismissal thing while I'm at it.

Reply:
> mbo to Fn+D for now and continuing the rest of the PR as is? Then we can follow up with extending the functionality to the text message frames later. :-)

will it be deleting both or just the frame it's on? if you do one click of it, and it deletes my waypoint and my message could leave a possibility of deleting an important message

Reply:
by choosing fn+w it focuses on just the way to delete, but possibility of stand alones to also set a way point in the future. Just a thought

Reply:
>will it be deleting both or just the frame it's on? if you do one click of it, and it deletes my waypoint and my message could leave a possibility of deleting an important message

I should be able to remove either a waypoint or a textmessage frame if I am currently on it in the frame index.